app/assets/stylesheets/wishlist.scss

```diff
@@ -0,0 +1,26 @@
+.wish-products-list {
+  height: 50em;
+}
+
+.border-wrap-grey-small {
+  border-bottom: 1px solid $grey-border;
+}
+
+.bigger-font {
+  font-size: 1.05em;
+}
```

Comment:
Nice suggestion @andela-ydaniju. The reason `border` was appended was because it's being used as a border color.